src/style.css

```
@@ -1 +1,39 @@
 /* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  align-items: center;
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url(./images/star-active.svg);
+}
```

```suggestion
.stars.stars--1 .stars__star:nth-child(-n + 1),
.stars.stars--2 .stars__star:nth-child(-n + 2) {
  background-image: url(./images/star-active.svg);
}
```
combine it with a comma